lib/http/2/huffman.rb

```diff
@@ -50,283 +43,282 @@ def decode(buf)
             #  [emit] character to be emitted on this transition, empty string, or EOS.
             #  [next] next state number.
             trans = MACHINE[state][branch]
-            trans.first == EOS and
-              raise CompressionError.new('Huffman decode error (EOS found)')
+            trans.first == EOS &&
+              fail(CompressionError, 'Huffman decode error (EOS found)')
             trans.first && emit << trans.first
             state = trans.last
           end
         end
         # Check whether partial input is correctly filled
         unless state <= MAX_FINAL_STATE
-          raise CompressionError.new('Huffman decode error (EOS invalid)')
+          fail CompressionError, 'Huffman decode error (EOS invalid)'
         end
-        emit.force_encoding(BINARY)
+        emit.force_encoding(Encoding::BINARY)
       end
 
       # Huffman table as specified in
       # - http://tools.ietf.org/html/draft-ietf-httpbis-header-compression-10#appendix-B
       CODES = [
```

Comment:
This table was aligned so that easy to cut-n-paste from the spec.
https://tools.ietf.org/html/draft-ietf-httpbis-header-compression-12#appendix-B